### PR TITLE
BZ#1998450: Updated procedure related to accessing registry metrics.

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.9+

Scope: Updated procedure related to accessing registry metrics.

[BZ1998450](https://bugzilla.redhat.com/show_bug.cgi?id=1998450)

Link to docs preview:
[Preview](https://56189--docspreview.netlify.app/openshift-enterprise/latest/registry/accessing-the-registry.html#registry-accessing-metrics_accessing-the-registry)

QE Review:
@wewang58 @xiuwang ptal

Additional Information:
[Accessing Registry Metrics](https://docs.google.com/document/d/1_FQFoKCn9IU0zBhxnZlTtd-44D31TfzLjce5iW3CuE0/edit)

### CODE REVIEW

File: modules/registry-accessing-metrics.adoc [hunk 1]
```diff
@@ -18,12 +18,47 @@ endpoint.
 
 .Procedure
 
-There are two ways in which you can access the metrics, running a metrics query
-or using the cluster role.
+There are two ways in which you can access the metrics, using the cluster role or running a metrics query.
```
Comment (xiuwang) [inline modules/registry-accessing-metrics.adoc:21]: Need update this description,
How about 
```There are a way in which you can access the metrics, running a metrics query using the cluster role``` ?

File: modules/registry-accessing-metrics.adoc [hunk 2]
```diff
@@ -18,12 +18,47 @@ endpoint.
...
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user prometheus-scraper <username>
+----
 
 *Metrics query*
 
-. Run a metrics query, for example:
+. Get user token.
```
Comment (xiuwang) [inline modules/registry-accessing-metrics.adoc:53]: how about `s/user/the user` ?

File: modules/registry-accessing-metrics.adoc [hunk 3]
```diff
@@ -56,44 +91,4 @@ imageregistry_http_request_duration_seconds{method="get",quantile="0.99"} 0.0159
 imageregistry_http_request_duration_seconds_sum{method="get"} 12.260727916000022
 ----
 <1> `<user>` can be arbitrary, but `<secret>` must match the value specified in the
```
Comment (xiuwang) [inline modules/registry-accessing-metrics.adoc:93]: s/but `<secret>` must match the value specified in the registry configuration./but `<secret>` must use the getting token

File: modules/registry-accessing-metrics.adoc (hunk 1 above)
Comment (subhtk) [reply modules/registry-accessing-metrics.adoc:21]: How about "To access the metrics, run a metrics query or use the cluster role."
or "You can access the metrics by running a metrics query or using a cluster role."

**Comment by xiuwang** [comment]
/lgtm
thanks

File: modules/registry-accessing-metrics.adoc [hunk 4]
```diff
@@ -18,12 +18,47 @@ endpoint.
...
-or using the cluster role.
+You can access the metrics by running a metrics query using a cluster role.
+
+*Cluster role*
+
+. Create a cluster role if you do not already have one to access the metrics:
++
+[source,terminal]
+----
+$ cat <<EOF | oc create -f -
```
Comment (kelbrown20) [inline modules/registry-accessing-metrics.adoc:29]: I think the render would look better if the sample output yaml block was separate from the command block. Similar to this: 

```
[source,terminal]
----
<command> 
----

.Example output
[source,yaml] 
----
<yaml>
----
```

File: modules/registry-accessing-metrics.adoc [hunk 5]
```diff
@@ -55,45 +90,4 @@ imageregistry_http_request_duration_seconds{method="get",quantile="0.9"} 0.01484
...
-+
-[source,yaml]
-----
-openshift:
-  version: 1.0
-  metrics:
-    enabled: true
-...
-----
+<1> `<user>` can be arbitrary, but `<secret>` must use the user token.
```
Comment (kelbrown20) [inline modules/registry-accessing-metrics.adoc:93]:
```suggestion
<1> The `<user>` object can be arbitrary, but `<secret>` tag must use the user token.
```

**Review by kelbrown20 — COMMENTED** [review]
Hello! Just some small formatting nits! Great work!

**Review by kelbrown20 — COMMENTED** [review]
New updates lgtm!

**Pull request merged**